src/cmake-tools.ts

```diff
@@ -152,6 +152,21 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
   private readonly _codeModel = new Property<codemodel_api.CodeModelContent|null>(null);
   private _codeModelDriverSub: vscode.Disposable|null = null;
 
+  private readonly _communicationModeSub = this.workspaceContext.config.onChange('cmakeCommunicationMode', () => {
```

Comment:
I don't really know what 'sub' means here.  I know it's reused from other places, but it doesn't really make sense to me.  Can you open an issue to rename some of these things?

Reply:
https://github.com/microsoft/vscode-cmake-tools/issues/1509